src/Compilers/CSharp/Portable/BoundTree/BoundExpressionExtensions.cs

```diff
@@ -14,6 +14,11 @@ public static bool IsLiteralNull(this BoundExpression node)
             return node.Kind == BoundKind.Literal && node.ConstantValue.Discriminator == ConstantValueTypeDiscriminator.Null;
         }
 
+        public static bool IsLiteralDefault(this BoundExpression node)
+        {
+            return node.Kind == BoundKind.DefaultOperator && (object)node.Type == null;
```

Comment:
Will `node.Type == null` in error situations such as `default(System)`? Should this method check syntax instead?